File: GridBeyondAssignmentSineadNaughton/Models/PriceCalculation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GridBeyondAssignmentSineadNaughton.Models
{
    public class PriceCalculation
    {
        public decimal PriceAverage { get; set; }

        public decimal PriceMax { get; set; }

        public decimal PriceMin { get; set; }

        public string MostExpensiveSixtyMinutesPeriod { get; set; }
    }
}

File: GridBeyondAssignmentSineadNaughton/ClientApp/src/app/models/PriceCalculation.ts
export class PriceCalculation {

  constructor(priceAverage: number, priceMax: number, priceMin: number, mostExpensiveSixtyMinutesPeriod: string) {
    this.priceAverage = priceAverage;
    this.priceMax = priceMax;
    this.priceMin = priceMin;
    this.mostExpensiveSixtyMinutesPeriod = mostExpensiveSixtyMinutesPeriod;
  }

  priceAverage: number;
  priceMax: number;
  priceMin: number;
  mostExpensiveSixtyMinutesPeriod: string;

}
